lib/plugins/aws/package/compile/events/alb/lib/target.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
+  getTargetId(functionName) {
+    const { targetAlias } = this.serverless.service.functions[functionName];
+    const lambdaLogicalId = this.provider.naming.getLambdaLogicalId(functionName);
+
+    return targetAlias
+      ? { Ref: targetAlias.logicalId }
```

Comment:
Why we do not reference it same way as here: https://github.com/serverless/serverless/blob/3910df1ba6a8b39367ce8d51adb90216251be2ba/lib/plugins/aws/package/compile/events/apiGateway/lib/permissions.js#L21

Is above proven to work?

Ideally it'll be good to maintain convention and produce same references across the stack (maybe reuse that util for all places in which we need such resolution)

Reply:
Intially I did, then I noticed `AWS::Lambda::Alias` returned value is actually the ARN of the lambda alias so I decided to use it since it avoid building it myself (and yes it works I tested it). 
But I agree that consistency is more important, I'll revert the change.